flake.nix

```diff
@@ -7,6 +7,28 @@
 
   outputs = { self, nixpkgs, utils }: utils.lib.eachDefaultSystem (system: let
     pkgs = import nixpkgs { inherit system; };
+    # TODO: need to be moved as flake input once this repository is a flake
+    gitignore = import (builtins.fetchTarball {
+      url = https://github.com/hercules-ci/gitignore/archive/647d0821b590ee96056f4593640534542d8700e5.tar.gz;
+      sha256 = "0ks37vclz2jww9q0fvkk9jyhscw0ial8yx2fpakra994dm12yy1d";
+    }) { inherit (pkgs) lib; };
+
+    sphinx = pkgs.python3.withPackages(p: [ p.sphinx ]);
```

Comment:
Use the environment provided by `mkPoetryEnv`, we don't want to be using multiple different versions of `sphinx`.
https://github.com/NixOS/nixops/blob/master/flake.nix#L13-L15